versions/3.1.0.md

```diff
@@ -2677,7 +2676,14 @@ components:
 
 When request bodies or response payloads may be one of a number of different schemas, a `discriminator` object can be used to aid in serialization, deserialization, and validation.  The discriminator is a specific object in a schema which is used to inform the consumer of the document of an alternative schema based on the value associated with it.
 
-When using the discriminator, _inline_ schemas will not be considered.
+The discriminator object is legal only when using one of the composite keywords `oneOf`, `anyOf`, `allOf`. Both _inline_ and references are valid child schemas.
```

Comment:
I am proposing to clarify **When using the discriminator, _inline_ schemas will not be considered.** I am not claiming my proposed change is necessarily correct AS IS. The problem is that we find the spec ambiguous. I wrote something in the hope that it can incrementally lead to a better wording of the spec.